spt/OrangeBox/scripts/framebulk_handler.cpp

```diff
@@ -207,6 +217,20 @@ namespace scripts
 					dataMap[std::make_pair(section, i)] = line[i];
 			}
 
+			bool isNoop = true;
+
+			for (size_t i = 0; i < line.size(); ++i)
+			{
+				if (line[i] != NOOP)
+				{
+					isNoop = false;
+					break;
+				}
+			}
```

Comment:
So what this loop is doing is marking the section as noop if it's comprised completely of the `<` character. So even something like `<<<<<<-` wouldn't pass as noop because there's one non-`<` character (and who knows how it'd parse later). I think what you meant was the inverse: `bool isNoop = false;` and then `if (line[i] == NOOP) { isNoop = true; break; }`.